### PR TITLE
ComunityModerators section added

### PR DESCRIPTION
**Pull request by veyselkartalmis**
ComunityModerators section added with ModeratorCard component

### CODE REVIEW

File: components/CommunityModerators/CommunityModerators.module.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.container {
+    @apply bg-[#050B1E] flex flex-col gap-[65px] justify-center items-center pt-20 pb-24 pl-4 pr-4;
```
Comment (OrhanOzkercin) [inline components/CommunityModerators/CommunityModerators.module.css]: `#050B1E` is actually defined as `primary-blue-400`

File: components/CommunityModerators/CommunityModerators.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const CommunityModerators = () => {
+    return (
+        <div className={styles.container}>
+            <div className={styles.titleContainer}>
+                <p className={styles.shadowTitle}>Topluluk Moderatörleri</p>
+                <p className={styles.mainTitle}>Topluluk Moderatörleri</p>
+                <p className={styles.desc}>Frontendship’in herkes için daha verimli ve eğlenceli bir topluluk olması için katkı sağlayan üyelerimiz.</p>
+            </div>
+            <div className={styles.cardContainer}>
+                <ModeratorsCard image={"https://images.pexels.com/photos/220453/pexels-photo-220453.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"} name={"Veysel Kartal"} title={"Frontend Developer"} />
```
Comment (OrhanOzkercin) [inline components/CommunityModerators/CommunityModerators.tsx]: You may want to use a loop here. Also I think images are only for example

File: components/CommunityModerators/ModeratorsCard/ModeratorCard.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from "./ModeratorCard.module.css";
+
+const ModeratorsCard = ({ image, name, title }: any) => {
```
Comment (OrhanOzkercin) [inline components/CommunityModerators/ModeratorsCard/ModeratorCard.tsx]: Don't we have a type for image, name, and title?

**Review by OrhanOzkercin — CHANGES_REQUESTED** [review]
Well done 👏 I wrote some comments. If you don't agree we may talk about it if you want

File: components/CommunityModerators/ModeratorsCard/ModeratorCard.tsx (hunk 3 above)
Comment (veyselkartalmis) [reply components/CommunityModerators/ModeratorsCard/ModeratorCard.tsx]: i fixed it

File: components/CommunityModerators/CommunityModerators.module.css (hunk 1 above)
Comment (veyselkartalmis) [reply components/CommunityModerators/CommunityModerators.module.css]: i fixed it

File: components/CommunityModerators/CommunityModerators.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styles from "./CommunityModerators.module.css";
+import ModeratorsCard from './ModeratorsCard/ModeratorCard';
+
+const CommunityModerators = () => {
+    return (
```
Comment (OrhanOzkercin) [inline components/CommunityModerators/CommunityModerators.tsx:29]: What about this kind of a solution? It feels more readable for me, what do you think?

```js
const CommunityModerators = () => {
const moderatorsCards = [
  {
    profileImage: 'linkX',
    name: 'nameX',
    title: 'titleX'
  },
  ...
]
    return (
        <div className={styles.container}>
            <div className={styles.titleContainer}>
                <p className={styles.shadowTitle}>Topluluk Moderatörleri</p>
                <p className={styles.mainTitle}>Topluluk Moderatörleri</p>
                <p className={styles.desc}>Frontendship’in herkes için daha verimli ve eğlenceli bir topluluk olması için katkı sağlayan üyelerimiz.</p>
            </div>
            <div className={styles.cardContainer}>
                // loop over moderatorsCards
                <ModeratorsCard  {...moderatorCardItem}/> // destruct a value of object
            </div>
        </div>
    )
}
```

File: components/CommunityModerators/CommunityModerators.tsx (hunk 2 above)
Comment (veyselkartalmis) [reply components/CommunityModerators/CommunityModerators.tsx]: I added the card array. I rewrote the card component code.

**Review by OrhanOzkercin — APPROVED** [review]
Excellent job 👏

**Pull request merged**